connect_cmd.go

```diff
@@ -127,77 +156,102 @@ func connectAction(ctx *cli.Context) error {
+					connectResult.InsightsError = errorMessages["insights"].message.Error()
+				} else {
+					fmt.Printf("  [%v] Analytics ... Cannot connect to Red Hat Insights\n", uiSettings.iconError)
+				}
+			} else {
+				connectResult.InsightsConnected = true
+				interactivePrintf("  [%v] Analytics ... Connected to Red Hat Insights\n", uiSettings.iconOK)
+			}
+			durations["insights"] = time.Since(start)
 		}
-		durations["insights"] = time.Since(start)
+	} else {
+		interactivePrintf(" [ ] Analytics ... Connecting to Red Hat Insights disabled\n")
 	}
 
-	/* 3. Start yggdrasil (rhcd) service */
-	if rhsmErrMsg, exist := errorMessages["rhsm"]; exist && rhsmErrMsg.level == log.LevelError {
-		connectResult.YggdrasilStarted = false
-		interactivePrintf(
-			"%v Skipping activation of %v service\n",
-			uiSettings.iconError,
-			ServiceName,
-		)
-	} else {
-		start = time.Now()
-		progressMessage := fmt.Sprintf(" Activating the %v service", ServiceName)
-		err = showProgress(progressMessage, activateService)
-		if err != nil {
+	if ManagementFeature.Enabled {
+		/* 3. Start yggdrasil (rhcd) service */
+		if rhsmErrMsg, exist := errorMessages["rhsm"]; exist && rhsmErrMsg.level == log.LevelError {
 			connectResult.YggdrasilStarted = false
-			errorMessages[ServiceName] = LogMessage{
-				level: log.LevelError,
-				message: fmt.Errorf("cannot activate %s service: %w",
-					ServiceName, err)}
-			if uiSettings.isMachineReadable {
-				connectResult.YggdrasilStartedError = errorMessages[ServiceName].message.Error()
+			interactivePrintf(
+				"  [%v] Skipping activation of %v service\n",
+				uiSettings.iconError,
+				ServiceName,
+			)
+		} else {
+			start = time.Now()
+			progressMessage := fmt.Sprintf(" Activating the %v service", ServiceName)
+			err = showProgress(progressMessage, activateService)
+			if err != nil {
+				connectResult.YggdrasilStarted = false
+				errorMessages[ServiceName] = LogMessage{
+					level: log.LevelError,
+					message: fmt.Errorf("cannot activate %s service: %w",
+						ServiceName, err)}
+				if uiSettings.isMachineReadable {
+					connectResult.YggdrasilStartedError = errorMessages[ServiceName].message.Error()
+				} else {
+					interactivePrintf("  [%v] Remote Management ... Cannot activate the %v service\n", uiSettings.iconError, ServiceName)
+				}
 			} else {
-				fmt.Printf("%v Cannot activate the %v service\n", uiSettings.iconError, ServiceName)
+				connectResult.YggdrasilStarted = true
+				interactivePrintf("  [%v] Remote Management ... Activated the %v service\n", uiSettings.iconOK, ServiceName)
 			}
-		} else {
-			connectResult.YggdrasilStarted = true
-			interactivePrintf("%v Activated the %v service\n", uiSettings.iconOK, ServiceName)
+			durations[ServiceName] = time.Since(start)
 		}
-		durations[ServiceName] = time.Since(start)
-		interactivePrintf("\nSuccessfully connected to Red Hat!\n")
+	} else {
+		interactivePrintf(" [ ] Management .... Starting %s service disabled\n", ServiceName)
```

Comment:
When I disabled the 'remote-management' feature, it looks like the space-indents
were missing one:

```
 [✓] Connected to Red Hat Subscription Management
  [✓] Content ... Red Hat repository file generated
  [✓] Analytics ... Connected to Red Hat Insights
 [ ] Management .... Starting yggdrasil service disabled
```

Reply:
Good catch. Matyas also reported this issue. :+1: